syncano/democracy/src/get-questions.js

```diff
@@ -0,0 +1,16 @@
+import Syncano from 'syncano-server'
+
+export default async ctx => {
+  const {response, data} = new Syncano(ctx)
+
+  try {
+    const questions = await data.question.list()
```

Comment:
Implement this algorithm https://stackoverflow.com/questions/3981314/hacker-news-algorithm-in-php#3981492